app.rb

```diff
@@ -22,7 +23,8 @@ def forward_request(request)
     primary_response, secondary_response = RequestForwarder.mirror_to(@primary, @secondary, request)
 
     # log comparison of the two responses
-    log_comparison ResponseComparator.compare(primary_response, secondary_response)
+    comparison_type = rand(100) <= @comparison_sample_pct ? :quick : :full
```

Comment:
If the `COMPARISON_SAMPLE_PCT` env variable isn't set, `@comparison_sample_pct` will be cast to 0, and this will default to the full comparison. 

Is that what we want, would it be safer to default to the quick comparison? 🤔

Reply:
you're right, I've changed that